code/sha3/Hacl.Impl.SHA3.fst

```diff
@@ -377,6 +377,28 @@ let storeState rateInBytes s res =
       copy res (sub block 0ul rateInBytes)
     )
 
+(* 
+   Lemma_equivalence_state_rate proves that it is equivalent
```

Comment:
it's `lemma_equiv_alloc_bigger_state`, no?

Reply:
:) Sure